src/main/java/com/microsoft/graph/models/extensions/Multipart.java

```diff
@@ -82,7 +173,7 @@ public void addPart(String name, String contentType, byte[] byteArray) throws IO
 	 * @throws IOException Throws an exception if the output stream cannot be written to
 	 */
 	public void addHtmlPart(String name, String content) throws IOException {
-		addPart(name, "text/html", content);
+		addFormData(name, "text/html", content.getBytes(MULTIPART_ENCODING));
```

Comment:
We can't make assumptions about the encoding of HTML content.  If we keep this addHtmlPart method, it needs to accept a byte array and not a string.

Reply:
Changed addHtmlPart to accept bytes.